storage/persistent-storage/persistent-storage-cinder.adoc

```diff
@@ -0,0 +1,22 @@
+[id="persistent-storage-cinder"]
+= Persistent storage using Cinder
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-cinder
+
+toc::[]
+
+<product-title} supports OpenStack Cinder. You can provision your {product-title} cluster with persistent storage using link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/8/html-single/architecture_guide/index#comp-cinder[OpenStack Cinder]. Some familiarity with Kubernetes and OpenStack is assumed.
```

Comment:
**<product-title}** should be **{product-title}**

Reply:
I'd also move the link into an **Additional references** section to be consistent with the other assemblies.